charts/ingress-nginx/values.yaml

```
@@ -844,6 +844,7 @@ controller:
     # if this port is changed, change healthz-port: in extraArgs: accordingly
     enabled: false
     service:
+      enabled: true
```

```suggestion
      # Enable the metrics service or not.
      enabled: true
```

Please re-run `helm-docs` after applying this.